### DISCUSSION TITLE
Launching PWAs with Mercury Browser

### DISCUSSION THREAD

**Discussion by onfaitr** [filips123/PWAsForFirefox#602] [Q&A]
Hi, I used to have it working before without touching anything, but since I reinstalled the PWAsForFirefox runtime when I launch any PWAs it's with the Firefox runtime and I want it to be the Mercury Browser runtime like before if it's possible...

(Sorry for my english)

EDIT : Here is the browser if you don't know https://github.com/Alex313031/Mercury

**Comment by filips123** [comment]
You have to download ZIP file of Mercury browser for your operating system, extract it into the [runtime directory](https://pwasforfirefox.filips.si/resources/installation-directories/#runtime) and symlink `mercury.exe`/`mercury` as `firefox.exe`/`firefox`.

To symlink it, run the following command inside the runtime directory:

* On Windows: `mklink firefox.exe mercury.exe`
* On Linux and macOS: `ln mercury firefox`
